### DISCUSSION TITLE
Location of PhotoPrism's Config files

### DISCUSSION THREAD

**Discussion by a-Rrrr** [tteck/Proxmox#2891] [Help Needed] [answered]
Hello, 

I used the PhotoPrism script to set it up on my proxmox server. Now i'm wondering where the config files are located, or what the new way to  configure the back end of PhotoPrism. Specifically the ymal/yml files. 

Thank you

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/blob/46be5a9b744cfc8691dab056f819572be84139c1/install/photoprism-install.sh#L43-L52

**Reply by a-Rrrr** [reply]
Thank you very much!

**Comment by ChristophCaina** [comment]
Hi, I need to ask the same - but I need to to change some of the settings in photoprism that are not available via the Ui...
https://docs.photoprism.app/user-guide/settings/advanced/

where should I find the config files?

**Reply by meifiant** [reply]
i add the config option parameter in /opt/photoprism/config/.env:

PHOTOPRISM_INDEX_SCHEDULE='@every 3h'
PHOTOPRISM_ORIGINALS_LIMIT='10000'
PHOTOPRISM_BACKUP_RETAIN='7'

after change i restart the LXC, and its up successfully.  however i still not able to check whether those change is effective or what.
